sources/packages/web/src/views/student/ApplicationStatus.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <Chip
```

Comment:
vue 3 `v-badge` was released, did u try that, https://next.vuetifyjs.com/en/components/badges/#api

Reply:
Just keep in mind the version of the Vuetify that we are using @ann-aot.
If we need to upgrade the version to use the new component we need to check if the basic functionalities are working and looking similar.
IMO, if it is available in the version that we are using, go for it, if we need to upgrade the Vuetify version, it is up to the developer.

Reply:
Did not try v-badge, can try in any of the upcoming stories related.